Library/Homebrew/keg.rb

```diff
@@ -339,14 +339,14 @@ def to_formula
   end
 
   def installed_dependents
+    return [] unless linked?
```

Comment:
Yeh, of course, thanks. I think checking the opt link is probably the best way of doing this and then opt-out (pun partially intended) of this safety feature if there's no `opt` link at all.